core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt

```diff
@@ -7,7 +7,7 @@ import org.json.JSONObject
  * Options used to configure the IFrame Player. All the options are listed here:
  * [IFrame player parameters](https://developers.google.com/youtube/player_parameters#Parameters)
  */
-class IFramePlayerOptions private constructor(private val playerOptions: JSONObject) {
+class IFramePlayerOptions private constructor(private val playerOptions: JSONObject, val origin: String) {
```

Comment:
Why is origin passed to the constructor? We should keep using the builder pattern instead of doing this.

Reply:
Do you think adding method in _IFramePlayerOptions_ instead of that would work?

`  fun getOrigin(): String {
        return playerOptions.getString(Builder.ORIGIN)
    }`

Reply:
Do you think adding method in _IFramePlayerOptions_ instead of that would work?

`  fun getOrigin(): String {
        return playerOptions.getString(Builder.ORIGIN)
    }`